src/app/component/Form/Form.component.js

```diff
@@ -118,6 +118,10 @@ class Form extends Component {
             const { current } = input;
             if (current && current.id && current.value) {
                 const { id, value } = current;
+                if (id === 'is_subscribed') {
```

Comment:
What does the specific input is doing in abstract component.

Reply:
Please check the type attribute here. Please extend this fix to radio buttons as well!